src/Utopia/Messaging/Adapters/Email/Mailgun.php

```diff
@@ -14,14 +14,25 @@ class Mailgun extends EmailAdapter
     public function __construct(
         private string $apiKey,
         private string $domain,
+        private bool $isUS = true
```

Comment:
Let's flip this to `$isEU`, people outside US might set this to false when not in EU